src/containers/CategoricalList.js

```diff
@@ -0,0 +1,168 @@
+import React, { Component } from 'react';
+import { compose, bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Flipper } from 'react-flip-toolkit';
+import cx from 'classnames';
+
+import { makeNetworkNodesForType, makeGetVariableOptions, makeGetPromptVariable, makeGetNodeDisplayVariable } from '../selectors/interface';
+import { actionCreators as sessionsActions } from '../ducks/modules/sessions';
+import { CategoricalItem } from '../components/';
+import { MonitorDragSource } from '../behaviours/DragAndDrop';
+import { getCSSVariableAsString } from '../utils/CSSVariables';
+import { getNodeAttributes, nodeAttributesProperty, nodePrimaryKeyProperty } from '../ducks/modules/network';
+
+const colorPresets = [
+  getCSSVariableAsString('--cat-color-seq-1'),
```

Comment:
Just something to flag here, I had problems with calling `getCSSVariableAsString()` for constants in Architect, I think because since we don't know if the DOM is ready when this code is called (i.e. variables may not be defined yet).

My solution was to call these methods at render time, though there might be a more elegant solution.

Reply:
Good to know! I can update that.